### ISSUE TITLE
How to change the progressWidget size?

### ISSUE THREAD

**Issue by buckydroid** [fayaz07/progress_dialog#82]
I'm currently trying to make a progress dialog for the signing up process.

` loadingDialog.style(
        message: "Signing up...",
        backgroundColor: Colors.grey[900],
        progressWidget: CircularProgressIndicator(),
        padding: EdgeInsets.all(20),
        elevation: 10.0,
        borderRadius: 3.0,
        messageTextStyle: TextStyle(color: Colors.white, fontSize: 21)
    );`

Everything works fine. But the CircularProgressIndicator is too big and looks kinda odd. Is there any ways to change the size of it?

Screenshot : https://imgur.com/a/xpu6uAH

**Comment by fayaz07** [comment]
I don't think you can do that directly, the api has to be updated in order to do that.
